### PR TITLE
CNV-31146: bug fix for uninstalling virt

### PR DESCRIPTION
**Pull request by sabrinajess**
Version(s): 4.11+

Issue: [CNV-31146](https://issues.redhat.com//browse/CNV-31146)

Link to docs preview: https://64780--docspreview.netlify.app/openshift-enterprise/latest/virt/install/uninstalling-virt#virt-deleting-virt-cli_uninstalling-virt

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/virt-deleting-virt-cli.adoc [hunk 1]
```diff
@@ -37,6 +37,13 @@ $ oc delete subscription kubevirt-hyperconverged -n openshift-cnv
 $ oc delete csv -n openshift-cnv -l operators.coreos.com/kubevirt-hyperconverged.openshift-cnv
 ----
 
+. Delete the {VirtProductName} namespace:
++
+[source,terminal]
+----
+$ oc delete namespace -n openshift-cnv
```
Comment (mburke5678) [inline modules/virt-deleting-virt-cli.adoc:44]: I think the command should be: ` oc delete namespace openshift-cnv:

```
$  oc delete namespace -n openshift-cnv
error: resource(s) were provided, but no name was specified
$  oc delete namespace openshift-cnv
namespace "openshift-cnv" deleted
```

Comment (mburke5678) [reply modules/virt-deleting-virt-cli.adoc:44]: I know it is outside of the scope of this PR. But, the [web console install instructions](https://docs.openshift.com/container-platform/4.13/virt/install/installing-virt-web.html) are a bit off. 

> 2. In the Filter by keyword field, type OpenShift Virtualization Operator.

The user needs to search for `OpenShift Virtualization`. If you search for `OpenShift Virtualization Operator`, nothing was coming up for me. 

image

File: modules/virt-deleting-virt-cli.adoc [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Module included in the following assemblies:
+mm// Module included in the following assemblies:
```
Comment (dbasunag) [inline modules/virt-deleting-virt-cli.adoc:1]: typo

**Pull request merged**